### ISSUE TITLE
flux 9.3.x | VariableViewHelper::getComponent() Error after Installation

### ISSUE THREAD

**Issue by chriskrj** [FluidTYPO3/flux#1739]
I installed flux 9.3 on an existing TYPO3 Project [9.5.13]. Frontend and Backend throws 
`Call to undefined method FluidTYPO3\Flux\ViewHelpers\Form\VariableViewHelper::getComponent()`

Plug and Play is Disabled and everything worked fine before update. 
My Template Files are located in my own Extension.

TYPO3 9.5.13
PHP 7.3
Flux 9.3.1

**Comment by chriskrj** [comment]
I use the VariableViewHelper in my Content Templates like this:

```html <flux:grid>
        <flux:grid.row>
          <flux:grid.column label="Col 1" name="col1" colPos="1" colspan="1">
            <flux:form.variable name="allowedContentTypes" value="{f:render(partial: 'Backend/AllowedContentTypes', section: 'FCETeaserElements')}"/>
          </flux:grid.column>
        </flux:grid.row>
      </flux:grid>```

**Comment by NamelessCoder** [comment]
@chriskrj Please provide a stack trace, then I will investigate.

**Comment by chriskrj** [comment]
Error Happens right after installation.
Install Tool is available, db compare adds two new columns.
Flush Cashes and reload Backend/Frontend. 
Error appears. Frontend and Backend the same.
 
```
Wed, 05 Feb 2020 14:36:03 +0100 [CRITICAL] request="4a45069b8db37" component="TYPO3.CMS.Core.Error.DebugExceptionHandler": Core: Exception handler (WEB): Uncaught TYPO3 Exception: Call to undefined method FluidTYPO3\Flux\ViewHelpers\Form\VariableViewHelper::getComponent() | Error thrown in file /../public/typo3conf/ext/flux/Classes/ViewHelpers/AbstractFormViewHelper.php in line 36. Requested URL: https://www.test.intern/typo3/index.php?route=%2Fmain&token=--AnonymizedToken-- - {"TYPO3_MODE":"BE","exception":{}}
```
```
at FluidTYPO3\Flux\ViewHelpers\AbstractFormViewHelper->callRenderMethod()
in /../www/vendor/typo3fluid/fluid/src/Core/ViewHelper/AbstractViewHelper.php line 250
```

**Comment by kurathor** [comment]
Removing the added callRenderMethod() from AbstractViewHelper fixes the issues I have with 9.3.x . The method requires every derived view helper to implement a getComponent() method which view helpers like VariableViewHelper do not possess.

I don't know why callRenderMethod() was added and which implications it has on the Fluid 3.0 compatibility. But I did take a quick look at the Fluid 3.0 code and it doesn't seem to be necessary, but I also haven't tested it with 3.0 yet.

**Comment by NamelessCoder** [comment]
@kurathor Although it works if removing the method, the idea behind adding it in the first place is to create a shorter execution path for all Form-related ViewHelpers on both 2.x and 3.x versions of Fluid. But you are completely correct that this method shouldn't assume that `getComponent` is implemented by all subclasses.

The right solution is to have this method call `renderStatic` which is compatible with all implementations of the abstract class (this also avoids having to refactor some ViewHelpers in a way that would constitute a breaking change).
